14/umbraco-workflow/getting-started/workflow-content-app.md

```diff
@@ -0,0 +1,99 @@
+
+### Content Approval Flow
+
+You can add different groups for different stages of content approval flow. Content Approval flow groups can be reordered via drag and drop. You can also apply the approval flow either for publish and unpublish workflow or only publish workflow.
+
+![Content approval flow](../../../10/umbraco-workflow/getting-started/images/content-approval-flow.gif)
+
+#### Approval Flow Types
+
+Approval Flows are available in three types: Content approval flow, Inherited approval flow, and Document type approval flow.
+
+A given content node may have all three approval flow types applied but only one will be applied as per the following order of priority:
+
+* **Content approval flow:** set directly on a content node via the **Configuration** section in the **Workflow** tab . This type will take priority over all others.
+* **Document type approval flow:** set in the **Settings** section. This approval flow will apply to all content nodes of the selected Document Type unless the node has a Content approval flow set. This feature requires a license.
+* **Inherited approval flow:** if a node has no Content approval flow set, nor a flow applied to its Document Type, Umbraco Workflow will traverse the content tree until it finds a node with a Content approval flow and will use this flow for the current change.
+
+![Approval Flow Types](../../../10/umbraco-workflow/getting-started/images/approval-flow-types.png)
+
+Review the current responsibilities for Approval Groups in the **Roles** tab of the **Approval Groups** section for **Node-based approvals** and **Document type approvals** only. For more information see the [Roles](../workflow-section/approval-groups.md#roles) section in the [Approval Groups](../workflow-section/approval-groups.md) article.
+
+![Approval Groups Roles](../../../10/umbraco-workflow/getting-started/images/approval-groups-roles.png)
+
+Document type approval flows can also include conditional stages i.e., only include **Translators** in the workflow when the **Description** property has changed. For more information on settings conditions in Document type approval flows, see the [Document type approval flows](../workflow-section/workflow-settings.md#document-type-approval-flows) section in the [Workflow Settings](../workflow-section/workflow-settings.md) article.
+
+Configuration cannot be modified when a content node is in a workflow process.
+
+#### Content reviews
+
+Content reviews is a tool that allows content editors to keep their content up-to-date. For more information, see the [Content reviews](../workflow-section/content-reviews.md) section.
+
+## History
+
+The History sub-section provides a chronological audit trail of workflow activity for the current node. It displays a table containing the following information:
+
+* Type of Publish.
+* Who the workflow is requested by.
+* The date the workflow was requested.
+* Comments.
+* Status of the workflow.
```

Comment:
I know I'm probably really knit-picking now, but the dot should only be used for lists with actual sentences, right?

```suggestion
* Type of Publish
* Who the workflow is requested by
* The date the workflow was requested
* Comments
* Status of the workflow
```